test/pretty-print.test.js

```diff
@@ -228,12 +328,54 @@ test('pretty print commonPrefix - handle constrained routes', t => {
   findMyWay.on('GET', '/test/:hello', { constraints: { version: '2.0.0' } }, () => {})
 
   const arrayTree = findMyWay.prettyPrint({ commonPrefix: false })
-  const arrayExpected = `└── / (-)
-    └── test (GET)
-        test (GET) {"host":"auth.fastify.io"}
-        └── /:hello (GET, PUT)
-            /:hello (GET) {"version":"1.1.2"}
-            /:hello (GET) {"version":"2.0.0"}
+  const arrayExpected = `\
+└── /test (GET)
+    /test (GET) {"host":"auth.fastify.io"}
+    └── /:hello (GET)
+        /:hello (PUT)
```

Comment:
Wasn't the GET,PUT collapse good?

Reply:
It's doable, but Idk if it's a good idea. For me, it's more clear when each line/branch represents each node. But I'm the wrong person to ask as I don't use pretty print at all.

Reply:
done